internal/cli/tiltfile_result.go

```diff
@@ -0,0 +1,87 @@
+package cli
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"os"
+
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	tiltanalytics "github.com/windmilleng/tilt/internal/analytics"
+	"github.com/windmilleng/tilt/internal/tiltfile"
+	"github.com/windmilleng/tilt/pkg/logger"
+	"github.com/windmilleng/tilt/pkg/model"
+)
+
+const TiltfileErrExitCode = 5
```

Comment:
picked this exit code arbitrarily as something that wasn't reserved for a specific meaning, but very open to suggestions on how i handle exit codes--the two cases I was trying to account for (a. something unexpected went wrong while trying to do the work, and b. there was a(n expected) error in the work) don't seem specifically addressed in e.g. [the guide nick linked](https://www.tldp.org/LDP/abs/html/exitcodes.html#EXITCODESREF).

Reply:
for what it's worth, I don't think the code fulfills this contract. In particular, "Tiltfile loading returned an Error" doesn't mean "this is an expected error". It's just not something you can guarantee at this level of the API

Reply:
separately, I think this kind of comment is better to put in the code than on the PR

Reply:
er that's fair, I didn't really mean expected vs unexpected, but I want the caller to be able to differentiate "something went wrong while trying to read the tiltfile" vs "got an error while executing the Tiltfile"